yakuake-init.sh

```diff
@@ -25,24 +25,55 @@ trim ()
     echo $var
 }
 
+# debugging helper function
+#
+# usage:
+# logm "There is an ERROR o.O!"
+#
+# view log:
+# $ tail -f /var/log/syslog
+#
+logm ()
+{
+    logger $(trim ${1})
```

Comment:
Mind adding a tag for `yaukuake-init` or something similar to identify all messages sourced from this script?